### PR TITLE
Fix link to and title of Webinars page

### PR DESCRIPTION
**Pull request by robmarcer**
## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `flowforge/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `flowforge/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Backport needed? -> add the `backport` label
 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

File: marketing/webinars.md [hunk 1]
```diff
@@ -1,7 +1,3 @@
-navTitle: Webinars
-navGroup: Sales & Marketing
----
-
```
Comment (joepavitt) [inline marketing/webinars.md:4]: @robmarcer why has this been removed? This is used to categorise the page on the left-side navigation

Comment (joepavitt) [reply marketing/webinars.md:4]: very strange, in the build webinars file it's not removing the breaker to merge the properties:

```
---
originalPath: marketing/webinars.md
updated: 2023-01-31 13:45:55 +0700
---
---
navTitle: Marketing
---
```

but it's only happening with this file...

Comment (joepavitt) [reply marketing/webinars.md:4]: Windows vs Mac/Linux problem...

The file was added with CRLF line endings, so our replace didn't catch it. @knolleary isn't there a way to configure a repo to always ensure consistent line endings?

**Comment by knolleary** [comment]
A quick google search suggests so - https://www.aleksandrhovhannisyan.com/blog/crlf-vs-lf-normalizing-line-endings-in-git

In the mean time, lets fix the script to handle the extra `\r`.

File: marketing/webinars.md (hunk 1 above)
Comment (knolleary) [reply marketing/webinars.md:4]: A quick google search suggests so - https://www.aleksandrhovhannisyan.com/blog/crlf-vs-lf-normalizing-line-endings-in-git

In the mean time, lets fix the script to handle the extra `\r`.

**Comment by knolleary** [comment]
There are a couple issues with the original file - one being the line endings, but we should reasonable just cope with that. Will fix the copy script to handle.

More crucially, it is missing the `---` on the very first line.

**Comment by joepavitt** [comment]
> More crucially, it is missing the --- on the very first line.

Even when I have that locally, it still wasn't working, but that would have been an issue previously.

**Comment by knolleary** [comment]
Fix for line ending handling - https://github.com/flowforge/website/pull/375

**Comment by joepavitt** [comment]
The line endings work has been fixed with:
- https://github.com/flowforge/handbook/pull/206
- https://github.com/flowforge/website/pull/375

**Pull request closed without merging**